Makefile

```diff
@@ -1,38 +1,20 @@
-PREFIX=/usr/local
-BINDIR=${PREFIX}/bin
-DESTDIR=
-BLDDIR = build
-BLDFLAGS=
-EXT=
-ifeq (${GOOS},windows)
-    EXT=.exe
-endif
+GOVERSION := go1.19.1
+SHELL := /bin/bash
+REPO_ROOT := $(shell pwd)
+SBIN_DIR ?= /usr/sbin
 
-APPS = nsqd nsqlookupd nsqadmin nsq_to_nsq nsq_to_file nsq_to_http nsq_tail nsq_stat to_nsq
-all: $(APPS)
+export PATH=${REPO_ROOT}/go/bin:$(shell printenv PATH)
+export GOPATH=${REPO_ROOT}/go
```

Comment:
Why you need to specify GOPATH explicitly?

Reply:
GOPATH specified in order to make sure that installed packages will be placed in go/bin